stan/math/gpu/kernels/basic_matrix_kernels.cl

```diff
@@ -0,0 +1,13 @@
+#define A(i,j)  A[j*rows+i]
```

Comment:
just curious: any reason this is macro'd instead of done directly? I know it's in the kernel code in the C-like language.

Reply:
We use the same type of macros in the more complex kernels that will follow, to increase readabilitity. Its also used here for consistentcy purposes.

Reply:
Thanks. That makes sense.